### PR TITLE
improve logging of conflicts

### PR DESCRIPTION
**Pull request by jdkandersson**
Improves the logging of conflicts to include new lines

### CODE REVIEW

File: src/check.py [hunk 1]
```diff
@@ -135,9 +135,13 @@ def _update_action_problem(action: UpdateAction) -> Problem | None:
         )
 
     logging.error(
-        "there is a problem preventing the execution of an action, action: %s, problem: %s",
+        (
+            "there is a problem preventing the execution of an action\npath: %s\nproblem: %s\n"
+            "action: %s"
```
Comment (weiiwang01) [inline src/check.py:140]: Can we break the long string at the point of newlines? For example
```python
   (
        "there is a problem preventing the execution of an action\n"
        "path: %s\n"
        "problem: %s\n"
        "action: %s"
   )

**Review by gregory-schiano — APPROVED** [review]
LGTM

File: src/check.py (hunk 1 above)
Comment (jdkandersson) [reply src/check.py]: Good point, fixed

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by weiiwang01 — APPROVED** [review]

**Pull request merged**